src/store.tsx

```diff
@@ -28,6 +31,7 @@ const store = configureStore({
     user: userSlice,
     inbox: inboxSlice,
     settings: settingsSlice,
+    gestures: gestureSlice,
```

Comment:
Probably change to "gesture" here too? Following "post" "comment" slice naming

Reply:
Fixed. But why isn't it `setting: settingSlice`? :)

Reply:
Technically, yeah :)

I suppose we could also rename to gestureSettings, but meh